src/lib/connection.ts

```diff
@@ -9,7 +9,11 @@ export interface TableMetadata {
   structure: string;
 }
 
-export type Columns = Array<{ name: string }>;
+export type Columns = Array<{
```

Comment:
Do we want to have a separate columns type for query responses vs the responses _with_ type and constraint data?

Reply:
Good call, I'll update now